ecosystem/sep-0010.md

```diff
@@ -6,8 +6,8 @@ Title: Stellar Web Authentication
 Author: Sergey Nebolsin <sergey@mobius.network>, Tom Quisel <tom.quisel@gmail.com>, Leigh McCulloch <@leighmcculloch>, Jake Urban <jake@stellar.org>
 Status: Active
 Created: 2018-07-31
-Updated: 2020-08-14
-Version 2.0.0
+Updated: 2020-??-??
```

Comment:
I'd just mark it as today. This field has always been funny to be since this doc lives on github.